README.md

```diff
@@ -149,7 +149,10 @@ end
 oneview_fc_network 'Fc1' do
   client <my_client>
   data <resource_data>
-  action [:create, :create_if_missing, :delete]
+  operation <op> # String. Used in patch action only. e.g., 'add'
```

Comment:
@tmiotto do we want to expose all these attributes to the end user? 
Or maybe have *add_scopes* & *remove_scopes* and treat *set_scopes* inside the create?

Reply:
Hmm, I see there's already something similar in the *switch* resource. So it is likely acceptable as is.